plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/Platform.kt

```diff
@@ -0,0 +1,13 @@
+package godot.gradle
+
+enum class Platform {
+    LINUX_X64,
+    WINDOWS_X64,
+    OSX_X64,
+    ANDROID_X32,
```

Comment:
We will need to check if we can even compile this. We hadn't had any 32bit targets because on 32bit we ran into out of memory exceptions because we were using to much ram for 32bit hosts to compile the project. But if i remember correctly one can build for 32bit android on x64 anyways but i'm not sure.
Anyways i'm fine with it for now.

Reply:
We could drop the `32` bit targets for now and just add them later on. Wdyt?